src/main/resources/messages_en.properties

```diff
@@ -370,6 +370,7 @@ transaction.notes.length=The max length of Transaction Notes is 255 characters
 transaction.initial.amount.required=Initial amount is required
 transaction.initial.amount.positive.value=Initial amount must be higher than zero
 transaction.wrong.transaction.type=Only Deposit transaction type is allowed
+transaction.wrong.transaction.status=Only Pending transaction status is allowed
```

Comment:
This message is not the more friendly one to show to the user. (previous one about Deposit will be deleted soon) 
This action is only allowed for Transactions with Pending Status. Please review. 

(I checked the AC and seems no message suggestion was provided)